packages/instantsearch.js/src/lib/utils/createSendEventForHits.ts

```diff
@@ -138,7 +138,7 @@ export function _buildEventPayloadsForHits({
           widgetType,
           eventType,
           payload: {
-            eventName,
+            eventName: eventName || 'Hit Converted',
```

Comment:
alternatively eventName could be required in the type when it's click or convert, but not sure if that can easily be done

Reply:
We could maybe have `sendEventForHits` satisfy both a custom payload signature and `type, hit, eventName`? For now though, having those defaults works for me.

Reply:
Yes, I prefer defaults too, as typescript types aren't enforced at runtime, meaning there will be more valid events.